plugins/modules/aci_fabric_pod_external_tep.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Samita Bhattacharjee (@samitab) <samitab@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_fabric_pod_external_tep
+short_description: Manage Fabric Pod External TEP (fabric:ExtRoutablePodSubnet)
+description:
+- Manage External TEP Fabric Pod Subnets.
+options:
+  pod_id:
+    description:
+    - The Pod ID for the External TEP.
+    type: int
+    aliases: [ pod ]
+  description:
+    description:
+    - The description for the External TEP.
+    type: str
+    aliases: [ descr ]
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+  pool:
+    description:
+    - The subnet IP address pool for the External TEP.
+    - Must be valid IPv4 or IPv6 and include the subnet mask.
+    - Example 192.168.1.0/24 or 2001:db8:abcd:0012::0/64
```

Comment:
I don't think we support v6 here. I think it is only v4.

Reply:
![image](https://github.com/CiscoDevNet/ansible-aci/assets/98932546/d89c28a0-32e3-4135-b656-54950b3a14ce)
Is the metadata incorrect here and it should be v4 only?

Reply:
However, IPv6 is valid input it doesn't appear that its actually supported. Removed IPv6 from here.